selenium_tests/base.py

```diff
@@ -62,3 +53,56 @@ def setup_blank_site(self):
             is_default_site=True,
             root_page=self.home
         )
+
+    # https://github.com/wagtail/wagtail/issues/1824#issuecomment-450575883
+    # Identical to TransactionTestCase._fixture_teardown except that 'allow_cascade' is
+    # forced.
+    def _fixture_teardown(self):
+        # Allow TRUNCATE ... CASCADE and don't emit the post_migrate signal
+        # when flushing only a subset of the apps
+        for db_name in self._databases_names(include_mirrors=False):
+            # Flush the database
+            inhibit_post_migrate = (
+                self.available_apps is not None or
+                (   # Inhibit the post_migrate signal when using serialized
+                    # rollback to avoid trying to recreate the serialized data.
+                    self.serialized_rollback and
+                    hasattr(connections[db_name], '_test_serialized_contents')
+                )
+            )
+            call_command('flush', verbosity=0, interactive=False,
+                         database=db_name, reset_sequences=False,
+                         allow_cascade=True,
+                         inhibit_post_migrate=inhibit_post_migrate)
+
+
+def create_remote_webdriver(preset: str = "chrome") -> webdriver.Remote:
+    if preset == "chrome":
+        options = webdriver.ChromeOptions()
+        options.add_argument('--ignore-ssl-errors=yes')
+        options.add_argument('--ignore-certificate-errors')
+        options.add_argument("--window-size=480,720")
+        options.add_argument("--start-maximized")
+        driver = webdriver.Remote(
+            command_executor=get_hub_url(),
+            desired_capabilities=DesiredCapabilities.CHROME,
+            options=options,
+        )
+    elif preset == "firefox":
```

Comment:
only the chrome node is available in the docker-compose, do we intend to add it later?

Reply:
I think it will have to be added later because some tests fail in Firefox, so more work is required to reconcile the differences between browsers. If a Firefox node is added to docker-compose now, it would not be used, so I've decided it's best to leave it out.